openreview/conference/templates/tabsConferenceWebfield.js

```diff
@@ -99,7 +99,8 @@ function load() {
   if (user && !_.startsWith(user.id, 'guest_')) {
     activityNotesP = Webfield.api.getSubmissions(WILDCARD_INVITATION, {
       pageSize: PAGE_SIZE,
-      details: 'forumContent,invitation,writable'
+      details: 'forumContent,invitation,writable',
+      sort: 'tmdate:desc'
```

Comment:
I thought you agreed that it didn't really matter what order the submissions were in on the venue homepage. I'm ok with having it be sorted in this way, but it would reduce the load on mongo to not sort.

Reply:
this query is to get the latest activity so I think we need to sort them here.

Reply:
Oh I see. In that case it looks good!